README.md

```diff
@@ -60,13 +62,48 @@ cd hpp-skel/
 #
 # This will:
 # - prompt you for the new name of your project and the new remote repo url
-# - automatically rename your local hpp-skel directory to the name of your project
-# - create a new branch called "hpp-skel-port"
-# - add, commit, and push
+# - automatically create a new directory for your new project repo
+# - create a new branch called "hpp-skel-port" within your new repo directory
+# - add, commit, and push that branch to your new repo
+
 ./scripts/liftoff.sh
 
 ```
 
+### Add custom code
+Once your project has ported hpp-skel, follow these steps to integrate your own code:
+
+- Create a dir in `./include` to hold your custom code. See the example code within [`./include`](https://github.com/mapbox/hpp-skel/tree/master/include) for reference.
+- Create a new file within your new directory, and add your custom method or class.
+- Create a module header file (see [hello_world.hpp](https://github.com/mapbox/hpp-skel/blob/master/include/hello_world.hpp)), and `#include` your new method or class. Make sure this file matches the name of the directory you created in the first step above.
+- Run `make` and see what surprises await on your new journey :boat:
+- If it compiles succesfully, congrats :tada: If not, dont fret. Take a breath and read the error message.
+- To start putting your header lib to work, setup a test to make sure it is working as expected. 
+
+### Setup tests
+Since header-only libraries are _not_ compiled, they require a compiled parent. This is done by using the [`#include`](https://github.com/mapbox/cpp/blob/master/glossary.md#include) keyword. We can use a compiled test file to use your new header, using [Catch](https://github.com/catchorg/Catch2) for assertions.
+
```

Comment:
Nice work starting to spell this out. The `parent` metaphor is a good idea, but I'm getting snagged on it, since I think the header would make more sense to be the "parent" since the .cpp depends on it. How about: `Since header-only libraries are _not_ normally compiled themselves, to test them you need to [`#include`](https://github.com/mapbox/cpp/blob/master/glossary.md#include) them in a `.cpp` (aka a [translation unit](https://github.com/mapbox/cpp/blob/master/glossary.md#translation-unit)) to compile and run their code. We recommend using [Catch](https://github.com/catchorg/Catch2) to make writing this `.cpp` file easy.

Reply:
Awesome, thanks @springmeyer ! Just pushed.

Reply:
✊  @GretaCB